.github/workflows/benchmark-runner.yml

```diff
@@ -0,0 +1,164 @@
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
+
+  benchmark-search-oss-cluster-08-primaries:
+    strategy:
+      matrix:
+        member_id: [1, 2, 3]
+    uses: ./.github/workflows/benchmark-flow.yml
+    secrets: inherit
+    with:
+      cluster_env: 1
+      benchmark_glob: "search*.yml"
+      module_path: bin/linux-x64-release/coord-oss/module-oss.so
+      allowed_envs: oss-cluster
+      allowed_setups: oss-cluster-08-primaries
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
+
+  benchmark-search-oss-cluster-16-primaries:
+    strategy:
+      matrix:
+        member_id: [1, 2, 3]
+    uses: ./.github/workflows/benchmark-flow.yml
+    secrets: inherit
+    with:
+      cluster_env: 1
+      benchmark_glob: "search*.yml"
+      module_path: bin/linux-x64-release/coord-oss/module-oss.so
+      allowed_envs: oss-cluster
+      allowed_setups: oss-cluster-16-primaries
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
+
+
+  benchmark-search-oss-cluster-20-primaries:
+    if: inputs.extended
+    strategy:
+      matrix:
+        member_id: [1]
+    uses: ./.github/workflows/benchmark-flow.yml
+    secrets: inherit
+    with:
+      cluster_env: 1
+      benchmark_glob: "search*.yml"
+      module_path: bin/linux-x64-release/coord-oss/module-oss.so
+      allowed_envs: oss-cluster
+      allowed_setups: oss-cluster-20-primaries
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
+
+
+  benchmark-search-oss-cluster-24-primaries:
+    if: inputs.extended
+    strategy:
+      matrix:
+        member_id: [1]
+    uses: ./.github/workflows/benchmark-flow.yml
+    secrets: inherit
+    with:
+      cluster_env: 1
+      benchmark_glob: "search*.yml"
+      module_path: bin/linux-x64-release/coord-oss/module-oss.so
+      allowed_envs: oss-cluster
+      allowed_setups: oss-cluster-24-primaries
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
+
+  benchmark-search-oss-standalone-profiler:
+    strategy:
+      matrix:
+        member_id: [1, 2, 3]
+    uses: ./.github/workflows/benchmark-flow.yml
+    secrets: inherit
+    with:
+      profile_env: 1
+      # TODO: change to "github-actions.profilers" when ready on grafana
+      triggering_env: "circleci.profilers" # results generated with profilers attached are not mixed with the ones without it
+      module_path: bin/linux-x64-release-profile/search/redisearch.so
+      benchmark_glob: "search*.yml"
+      benchmark_runner_group_member_id: ${{ matrix.member_id }}
+      benchmark_runner_group_total: ${{ strategy.job-total }}
+
+
+  benchmark-vecsim-oss-standalone-profiler:
```

Comment:
Perhaps we should put the profilers under `extended` as well? and maybe the 16 primaries run? to reduce resources as it run for every PR. Then, we can consider adding a trigger for the extended run by label (and/or decide the granularity)

Reply:
Currently, I kept the existing behavior. We can redefine stuff but IMO we can do that later, the goal of this PR is not to redefine the flow